infrastructure/demo/variables.tf

```
@@ -0,0 +1,54 @@
+variable "project_name" {
+  description = "value for the project name"
+  type        = string
+}
+
+variable "environment" {
+  description = "value for the environment"
+  type        = string
+}
+
+variable "billing_code" {
+  description = "value for the billing code"
+  type        = string
+}
+
+variable "resource_tags" {
+  description = "map of tags to be applied to all resources"
+  type        = map(string)
+  default     = {}
+
+}
+
+variable "aws_region" {
+  description = "AWS region to deploy to"
+  type        = string
+  default     = "af-south-1"
+
+}
+
+variable "demo_profile" {
+  description = "AWS profile to use for development"
+  type        = string
+  default     = "aaq_demo"
```

Should this be
```suggestion
  default     = "aaq"
```
or does it not matter? e.g. would these be overridden by `demo.auto.tfvars`?

---

Answer: It will get overridden, but I'll commit this for consistency